PBM-1430 Added about a docs page (#238)
percona/pbm-docs: docs/about-docs.md

# About Percona Backup for MongoDB documentation

In this documentation you will find comprehensive information about the features, operations, and functionality of Percona Backup for MongoDB (PBM). Also reference materials on PBM commands and configuration file options are available for you here.

## Versions

> We encourage our users to run the latest version of PBM so that they receive all bug fixes and can enjoy all new features and improvements available with the latest version. For this reason, the documentation always reflects the latest PBM version. 

Reviewer: ```suggestion
You should run the latest version of PBM to receive all bug and security fixes and benefit from the new features. For this reason, the documentation always reflects the latest PBM version. 
```

> To track the changes in features, we indicate the version where a feature appeared or a change was introduced. Check the release notes for detailed information.

Reviewer: ```suggestion
To track feature changes, an additional version indication is added where a feature appeared or a change was introduced. Please take a look at the release notes for detailed information.
```

## Report an issue or request a change

To report an issue with the documentation, leave your feedback using the widget at the bottom of the page or open a ticket in [Percona Backup for MongoDB Jira project :octicons-link-external-16:](https://perconadev.atlassian.net/jira/software/c/projects/PBM/issues).

## How to contribute

> Thank you for willing to contribute to Percona Backup for MongoDB project. Please check the [Contribution guidelines](reference/contributig.md) for  instructions.

Reviewer: ```suggestion
Thank you for your willingness to contribute to Percona Backup for MongoDB project. Please check the [Contribution guidelines](reference/contributing.md) for instructions.
```

Reviewer: + we need to rename the "contributig.md" file ;)
